cmd/fleetctl/get.go

```diff
@@ -1095,12 +1088,10 @@ func getMDMAppleCommand() *cli.Command {
 			warnDate := time.Now().Add(expirationWarning)
 			if mdm.RenewDate.Before(time.Now()) {
 				// certificate is expired, print an error
-				log(c, "\n")
-				printColoredMsg(c, "ERROR: Your Apple Push Notification service (APNs) certificate is expired. MDM features are turned off. To renew your APNs certificate, follow these instructions: [TODO link to documentation]", tablewriter.FgHiRedColor)
+				color.New(color.FgRed).Fprintln(writerOrStdout(c.App.Writer), "\nERROR: Your Apple Push Notification service (APNs) certificate is expired. MDM features are turned off. To renew your APNs certificate, follow these instructions: [TODO link to documentation]")
```

Comment:
nitpick: using `writerOrStdout` is redundant. This is because a default `Writer` is set when calling `cli.NewApp()`, and is overridden afterwards in `createApp`. We should consider removing it and all usages from `cmd/fleetctl/get.go`. Most commands in `cmd/fleetctl` use `c.App.Writer` directly.

Reply:
Gotcha, I'll remove it in a subsequent PR (I have one in progress so I'll do it as part of it).